docs/rofl/prerequisites.md

```diff
@@ -77,20 +98,18 @@ nightly-2022-08-22-x86_64-unknown-linux-gnu (overridden by '/code/rust-toolchain
 rustc 1.65.0-nightly (c0941dfb5 2022-08-21)
 ```
 
-For testing ROFL binaries on Sapphire Localnet, the binaries should be compiled
-for [MUSL C standard library]. You will need to add the following target to your
-rust environment:
+Make sure you have the correct target for rust to compile for:
 
 ```shell
-rustup target add x86_64-unknown-linux-musl
+rustup target add x86_64-unknown-linux-gnu
 ```
 
-Additionally, you will need the MUSL wrapper for gcc, the multilib package and
-clang for compiling the `mbedtls-sys-auto` dependency. On Ubuntu/Debian systems,
+In addition, you will need gcc's multilib support package and clang for
+compiling the `mbedtls-sys-auto` dependency. On Ubuntu/Debian systems,
 you can install those by running:
 
 ```shell
-sudo apt install musl-tools gcc-multilib clang
+sudo apt install gcc-multilib clang
```

Comment:
Note: when trying to run it in ubuntu docker ([and devcontainer](https://github.com/oasisprotocol/oasis-sdk/blob/lw/codespace/.devcontainer/init.sh#L11)), it also required:
```sh
sudo apt update
sudo apt -y install pkg-config protobuf-compiler cmake
```